ipn/ipnserver/server.go

```diff
@@ -659,7 +660,7 @@ func (s *Server) writeToClients(n ipn.Notify) {
 // The getEngine func is called repeatedly, once per connection, until it returns an engine successfully.
 //
 // Deprecated: use New and Server.Run instead.
-func Run(ctx context.Context, logf logger.Logf, ln net.Listener, store ipn.StateStore, linkMon *monitor.Mon, dialer *tsdial.Dialer, logid string, getEngine func() (wgengine.Engine, error), opts Options) error {
+func Run(ctx context.Context, logf logger.Logf, ln net.Listener, store ipn.StateStore, linkMon *monitor.Mon, dialer *tsdial.Dialer, logid string, getEngine func() (wgengine.Engine, *netstack.Impl, error), opts Options) error {
```

Comment:
Chonky argv

(which, if I understood it correctly, is something the Parts may help with so we don't have to pass all of these every time?)

Reply:
yes but also: this whole `ipnserver.Run` entry point is deprecated and only used by Windows.

Windows only uses it because a bunch of things on Windows sporadically fail on start-up due to various async net races inside the Windows kernel so we have to keep retrying until things are happy.

We really need to move the retries around (or figure out what to actually wait for) so we can get rid of `ipnserver.Run` entirely.